packages/typography/sass/font-brand.scss

```diff
@@ -6,7 +8,6 @@
 	url( 'https://s1.wp.com/i/fonts/recoleta/400.woff' ) format( 'woff' );
 }
 
-$serif: 'Noto Serif', Georgia, 'Times New Roman', Times, serif;
 $brand-serif: 'Recoleta', $serif;
```

Comment:
The `font-family.scss` file would be a better home for this variable definition.

Reply:
Kinda, yeah. It does fit here too. I was torn between this and the family-family but I have no strong opinion where it is. Happy to move it over. 👍